lib/rollbar/request_data_extractor.rb

```diff
@@ -28,7 +28,12 @@ def extract_request_data_from_rack(env)
       session = rollbar_filtered_params(sensitive_params, rollbar_request_session(rack_req))
       route_params = rollbar_filtered_params(sensitive_params, rollbar_route_params(env))
 
-      params = request_params.merge(get_params).merge(post_params).merge(raw_body_params)
+      params =
+        if !raw_body_params.empty?
```

Comment:
After making my changes, **without** L31-36 above, I eventually found that `params` was getting the post params twice: once as the desired hash and once in some serialized form attached to a key called `key`, IIRC. This seems to be because the `post_params` contained this undesirable value while the `raw_body_params` had what I needed to get the new tests passing while not breaking the old tests.